### PR TITLE
Topic3

### PR DESCRIPTION
**Pull request by DiegoJStroppiana**
Here is exercise 4 and 5 of Topic 3

### CODE REVIEW

**Comment by DiegoJStroppiana** [comment]
Sorry, I had an error in the comment. Here is ALL exercises of Topic 3

File: bootCampTopic3/bootCampTopic3Ex123/src/main/java/maven/exercise/bootCampTopic3Ex123/ServiceShoppingCartImp.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	/**
+	  * This is a service which allows to add an item, delete one existing item and show
+	  * actual items in a shopping cart
+	  *@author Diego Jesús Stroppiana
+	  *@version 1.0
+	  *@see ShoppingCart
+	  */
+	
+	public void addItem(Item i) {
+		// TODO Auto-generated method stub
```
Comment (gastonabastoG) [inline bootCampTopic3/bootCampTopic3Ex123/src/main/java/maven/exercise/bootCampTopic3Ex123/ServiceShoppingCartImp.java]: You must delete this comment, the TODO comment

File: bootCampTopic3/bootCampTopic3Ex123/src/main/java/maven/exercise/bootCampTopic3Ex123/ShoppingCart.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+	public void setItems(ArrayList<Item> items) {
+		ShoppingCart.items = items;
+	}
+	
+	public void addItem (Item i){
+		items.add(i);
+	}
+	
+	public void deleteItem (String name){
```
Comment (gastonabastoG) [inline bootCampTopic3/bootCampTopic3Ex123/src/main/java/maven/exercise/bootCampTopic3Ex123/ShoppingCart.java]: In this method you can retur a boolean value, when the method is succefull or not

File: bootCampTopic3/bootCampTopic3Ex123/src/test/java/maven/exercise/bootCampTopic3Ex123/ServiceShoppingCartImpTest.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import org.junit.Test;
+
+
+public class ServiceShoppingCartImpTest {
+	
+	static ServiceShoppingCartImp SSCI;
+	static ServiceShoppingCartImp SSCI2;
+	
+	
+//    @BeforeClass
```
Comment (gastonabastoG) [inline bootCampTopic3/bootCampTopic3Ex123/src/test/java/maven/exercise/bootCampTopic3Ex123/ServiceShoppingCartImpTest.java]: Remember to delete comments

File: bootCampTopic3/bootCampTopic3Ex45/src/main/java/maven/exercise/bootCampTopic3Ex45/UserCRUDServiceImp.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+		/**
+		  * This method updates a user from the list
+		  * @param u This parameter is the user which will be modified from the list with new data
+		  */
+		
+		UserDB.getInstance().updateUser(u);
+	}
+
+	public void showUserList() {
+		// TODO Auto-generated method stub
```
Comment (gastonabastoG) [inline bootCampTopic3/bootCampTopic3Ex45/src/main/java/maven/exercise/bootCampTopic3Ex45/UserCRUDServiceImp.java]: Remember to delete all TODO comments

**Comment by gastonabastoG** [comment]
This branch has conflicts that must be resolved, we have to resolve this

**Pull request merged**